features/OpenWebSite.feature

```diff
@@ -3,5 +3,5 @@ Feature: Posobolity open the website
   Scenario: As a user, I can open the website and see the main page
 
     Given Open www.booking.com website
-    When Click on "Accept cookie" button
+    When Click on "Main Page > Accept cookie button"
```

Comment:
Are you going to parse the string?

Reply:
![image](https://user-images.githubusercontent.com/87827778/223463219-f47cda53-cdfa-4eef-8a3b-545d8cf7ee80.png)
I don`t know why you see the old code